src/server/fedavg.py

```diff
@@ -559,6 +642,7 @@ def aggregate(self, client_packages: OrderedDict[int, dict[str, Any]]):
                 aggregated = torch.sum(client_params * weights, dim=-1)
 
                 global_param.data = aggregated
+        self.model.load_state_dict(self.public_model_params)
```

Comment:
`self.model.load_state_dict(self.public_model_params, strict=False)`, due to when `buffers` set as `local`, `self.public_model_params` will not have keys of buffers.

Reply:
Ah yes, I missed this, thanks for the pointer! I also thought about reinitializing things like batchnorms before assigning the state dict so that in case that buffers are not send, it is possible to start from a clean slate each round. I saw some people deal with batchnorms by evaluating the models in train mode centrally, which is easier imo from a clean batchnorm layer.